CONTRIBUTING.md

```diff
@@ -28,6 +28,23 @@ Here are a few things you can do that will increase the likelihood of your pull
 * Keep your change as focused as possible. If there are multiple changes you would like to make that are not dependent upon each other, consider submitting them as separate pull requests.
 * [Write a good commit message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
 
+### Unity & LTS support
+
+As we are a group of volunteers, we can only officially support the versions of Unity that are actively supported by Unity themselves. Please check the official [LTS FAQ](https://support.unity.com/hc/en-us/articles/4403332003348-What-is-a-Unity-LTS-Long-Term-Support-version-and-what-can-I-expect-from-it-) for more up-to-date information.
+
+To recap, we currently support:
```

Comment:
much like with my earlier comment on the PR this just looks a bit specific and detailed.
It feels a lot like the sort of line we forget to remove and then have people confused.
I think something like "non-preview release of Unity from 2019.4 and above" then we only have to keep track of a single line.

I guess its more we don't explicitly support anything but what the current mainline versions of Unity are, which varies more than we'd like.